lintreview-env

```diff
@@ -1 +1,3 @@
 LINTREVIEW_GUNICORN_BIND=0.0.0.0:5000
+LINTREVIEW_GUNICORN_LOG_ACCESS=-
```

Comment:
I had to do this locally too, to get the output logging to stdout.
:+1: to this.